test/testgarbage.cpp

```diff
@@ -1771,6 +1771,9 @@ class TestGarbage : public TestFixture {
             "void f() {\n"
             "    auto fn = []() -> foo* { return new foo(); };\n"
             "}\n");
+	
+	checkCode(
```

Comment:
Please move this test. This code is not garbage. It should be added in `TestTokenizer::findGarbageCode()`.